packit/api.py

```diff
@@ -800,9 +804,11 @@ def _run_kinit(self) -> None:
             logger.info("Won't be doing kinit, no credentials provided.")
             return
 
+        stg = "STG." if self.stage else ""
+
         cmd = [
             "kinit",
-            f"{self.config.fas_user}@FEDORAPROJECT.ORG",
+            f"{self.config.fas_user}@{stg}FEDORAPROJECT.ORG",
```

Comment:
Could this be configured as `{self.config.kerberos_realm}` for example, instead of being computed here?

This could make the whole setup more flexible, as the config could be change to match some other environment (which might be something other than "stg").

Reply:
+1, that should be more flexible, especially if we go into other envs, not just fp.o

Reply:
added a field to user config